.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+dist: trusty
+
+cache:
+  directories:
+    - client/node_modules
+
+branches:
+  only:
+  - master
+  - staging
+  - stable
+
+before_install:
+  - cp .env.example .env
+  - cp client/ci.karma.conf.js client/karma.conf.js
+  - nvm install 6.9.1
+  - nvm use 6.9.1
+  - npm install -g yarn
+
+install:
+  - yarn
+
+before_script:
+  - export CHROME_BIN=chromium-browser
+#  - "export DISPLAY=:99.0"
+#  - "sh -e /etc/init.d/xvfb start"
+
+script:
+  - ./bin/ci
+  - ./bin/ci_karma
```

Comment:
This runs the karma specs, obviously, and that file also needs to be added to this PR.